File: debugging-questions/Round1/py/question9.py
def are_anagrams(str1, str2):
    
    if len(str1) != len(str2):
        return False
    for char in str1:
        if char not in str2:
            return False  
    return True
str1 = "Santa"
str2 = "satan"
if are_anagrams(str1, str2):
    print(f"'{str1}' and '{str2}' are anagrams!")
else:
    print(f"'{str1}' and '{str2}' are not anagrams.")

File: debugging-questions/Round1/c/question9.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>
int are_anagrams(char *str1, char *str2) {
    if (strlen(str1) != strlen(str2)) {
        return 0;
    }
    for (int i = 0; i < strlen(str1); i++) {
        if (strchr(str2, str1[i]) == NULL) {
            return 0;  
        }
    }
    return 1;
}
int main() {
    char str1[] = "Santa";
    char str2[] = "satan";
    if (are_anagrams(str1, str2)) {
        printf("'%s' and '%s' are anagrams!\n", str1, str2);
    } else {
        printf("'%s' and '%s' are not anagrams.\n", str1, str2);
    }
    return 0;
}
